package.json

```diff
@@ -0,0 +1,133 @@
+{
+  "name": "codacy-eslint",
+  "description": "Eslint Tool for Codacy",
+  "keywords": [
+    "codacy",
+    "eslint",
+    "javascript",
+    "linter",
+    "typescript"
+  ],
+  "homepage": "https://github.com/codacy/codacy-eslint#readme",
+  "bugs": {
+    "url": "https://github.com/codacy/codacy-eslint/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/codacy/codacy-eslint.git"
+  },
+  "license": "Apache-2.0",
+  "main": "dist/src/index.js",
+  "directories": {
+    "src": "./src",
+    "test": "./src/test"
+  },
+  "scripts": {
```

Comment:
I'm running directly from node. Since it requires files in special places "/src" directory "/.codacyrc" file, I don't see value in running it outside of the docker container.

Reply:
:+1: